### ISSUE TITLE
[Bug]: Luminosity ratio of the focus indicator on the 'Page size' drop-down option with its background is less than the required ratio in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#10398]
### Link to reproducible scenario

https://www.ag-grid.com/javascript-data-grid/row-pagination/

### Describe the bug

Steps to reproduce 

1. Open the URL: [JavaScript Grid: Row Pagination | AG Grid](https://www.ag-grid.com/javascript-data-grid/row-pagination/) in latest edge browser.
2. 'Row Pagination' page will appear.
3. Press tab key to move to the 'Page size' drop-down control and activate it using enter key. drop-down options will appear with focus on the selected item. Take a screenshot and open it.
4. Open Color contrast Analyser tool and select forground and background color of focus indicator.
5. Verify whether Luminosity ratio of the focus indicator on the 'Page size' drop-down option with its background is less than the required ratio or not.

Actual behaviour
Luminosity ratio of the focus indicator on the 'Page size' drop-down option with its background is 1.084:1 which is less than the required ratio in Windows.

https://github.com/user-attachments/assets/31deba49-7458-4150-af4c-3def1c887d96

![Image](https://github.com/user-attachments/assets/79393021-9671-4d1b-aa60-c3e13c32c6f1)

Expected behaviour
Luminosity ratio of the focus indicator on the 'Page size' drop-down option with its background should be greater than or equal to 3:1 ratio in Windows.

More information
**Test Environment:**
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Edge Version 135.0.3179.73 (Official build) (64-bit)
URL: [JavaScript Grid: Row Pagination | AG Grid](https://www.ag-grid.com/javascript-data-grid/row-pagination/)

**User Impact:**
Low vision users are not able to identify the currently focused control if focus indicator is not visible clearly on receiving focus.

### Version

30.1.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for the suggestion.

Currently there's no way to implement this behaviour with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference: **AG-6163**

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

You can follow the status of this item on the product pipeline page::
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
